frontend/openchat-client/src/openchat.ts

```diff
@@ -326,6 +326,7 @@ export class OpenChat extends EventTarget {
     private _lastOnlineDatesPromise: Promise<Record<string, number>> | undefined;
     private _cachePrimer: CachePrimer | undefined = undefined;
     private _membershipCheck: number | undefined;
+    private _referralCode: string | undefined = undefined;
```

Comment:
We've called this `referredBy` everywhere else (FE and BE) so I think it should be the same here.